src/main/java/com/mraof/minestuck/skaianet/Session.java

```diff
@@ -27,6 +28,8 @@
 	
 	final Map<PlayerIdentifier, PredefineData> predefinedPlayers;
 	final Set<SburbConnection> connections;
+	final GristGutter gutter;
```

Comment:
The gutter is not currently being serialized, which is a problem. It is however fine to wait with that until the gutter itself is more finalized.